README.md

```diff
@@ -22,18 +20,65 @@ Dedicated to making your workflow flow more seamless. Search everything from app
 
 <p align="center"> <sub>Remember to star it, flow will love you more :)</sub></p>
 
-<p align="center"><img width="12px" src="https://user-images.githubusercontent.com/26427004/104119722-9033c600-5385-11eb-9d57-4c376862fd36.png"> <a href="https://www.softpedia.com/get/System/Launchers-Shutdown-Tools/Flow-Launcher.shtml">SOFTPEDIA EDITOR'S PICK</a></p>
-
 <img src="https://user-images.githubusercontent.com/6903107/144858082-8b654daf-60fb-4ee6-89b2-6183b73510d1.png" width="100%">
 
 
-## 🎉 New Features in 1.9
+## 🎅 New Features🤶
+### Preview Panel
+<img src="https://user-images.githubusercontent.com/6903107/207159213-662999d3-2c18-4256-b473-c417efca0069.png" width="400">
+
+- You can use the F1 key to open/hide the preview panel.
+- Media files display large images.
+- If it is not a media file, a large icon and the entire path are displayed.
+- You can turn it on permanently via Settings (Always Preview).
+- Added hotkeys (Ctrl+Plus,Minus / Ctrl+],[) to quickly change flow window's width and height. If you feel that the preview area is narrow, adjust it quickly.
+- This feature is currently in its early stages.
```

Comment:
```suggestion
- Use the F1 key to open/hide the preview panel.
- Media files will be displayed as large images, otherwise a large icon and entire path will be displayed.
- Turn on preview permanently via Settings (Always Preview).
- Use hotkeys (Ctrl+Plus,Minus / Ctrl+],[) to adjust flow's search window width and height quickly if the preview area is too narrow.
- This feature is currently in its early stages.
```